src/main/java/org/thoughtcrime/securesms/notifications/NotificationCenter.java

```diff
@@ -535,7 +541,7 @@ public void addNotification(int accountId, int chatId, int msgId) {
                     .setContentText("New messages") // content text would only be used on SDK <24
                     .setContentIntent(getOpenChatlistIntent(accountId));
                   if (privacy.isDisplayContact()) {
-                    summary.setSubText(accountAddr);
+                    summary.setSubText(accountTag);
```

Comment:
same here:

```suggestion
                if (!TextUtils.isEmpty(accountTag)) {
                    builder.setSubText(accountTag);
                }
```

(could be combined with the outer `if`, this seems not possible on github)